src/pages/Patch.tsx

```diff
@@ -65,6 +65,16 @@ const PatchCore: React.FC = () => {
         hasData={!!patch}
         title={description || `Patch ${get(patch, "patchNumber")}`}
         badge={<PatchStatusBadge status={status} />}
+        buttons={
+          <ActionButtons
+            canAbort
+            canRestart
+            canSchedule
+            canUnschedule
+            canSetPriority
+            initialPriority={1}
```

Comment:
i'm confused. where are these coming from?

Reply:
These are the required props for the `ActionButtons` component on the patch page. Since their functionality isn't set up i just passed in these props as true

Reply:
I think these are just placeholder props from the ActionButtons in the task page. The contents of ActionButtons will have to reflect the actions we can perform on a patch. This is the closest ticket that describes the task: https://jira.mongodb.org/browse/EVG-7353